lib/piezo.js

```diff
@@ -355,4 +423,9 @@ Piezo.prototype.stop = function() {
   return this;
 };
 
+if (IS_TEST_MODE) {
+  Piezo.frequencyToTone = frequencyToTone;
```

Comment:
You can use this same silliness if you want. It's "meh", but works and stays out of the way of non-test-runtime API